src/api/instance-methods.md

```diff
@@ -32,7 +36,17 @@
         // do something
       })
 
-      // function
+      // function for watching a deep path
+      this.$watch(
+        // Only `this.c.d` will be watched
```

Comment:
**nitpick**: I believe with the change of code comment above, we could simply remove this explanation

Reply:
👍 I removed the redundant comments.